.github/workflows/diff-report.yml

```diff
@@ -227,19 +227,17 @@ jobs:
       - name: Set output
         id: out
         run: |
-          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
-          {
-            echo "message<<$EOF"
-            cat message
-            echo "$EOF"
-          } >> "$GITHUB_OUTPUT"
+          cd checkstyle
+          ./.ci/append-to-github-output.sh "message" "$(cat ../message)"
```

Comment:
Let's handle `cat` in script

Reply:
I am ok to keep script to deal with multiple line variables

Reply:
Idea of the script was to make yaml as simple as possible and keep all details inside script with comments and descriptive variable names.

Reply:
If this was  original intend, we need to move all code out to shell script , not a part.

Reply:
I'd say it belongs in a separate file because it is the exact same code in both `site` and `diff-report`.